src/main.cpp

```diff
@@ -2437,6 +2599,40 @@ static bool ApplyTxInUndo(const CTxInUndo& undo, CCoinsViewCache& view, const CO
     return fClean;
 }
 
+/**
+ * Apply the undo operation of a CTzeInUndo to the given chain state.
+ * @param undo The undo object.
+ * @param view The coins view to which to apply the changes.
+ * @param out The out point that corresponds to the tx input.
+ * @return True on success.
+ */
+static bool ApplyTzeInUndo(const CTzeInUndo& undo, CCoinsViewCache& view, const COutPoint& out)
```

Comment:
This isn't being called by anything, I think it should be being called by `DisconnectBlock` right?

Reply:
Good catch.

Reply:
Fixed in dc3ff8300c1eaa12b8e650a09c74ba14c43041fc - please verify.

Reply:
Correction: dc3ff83 was a partial resolution with a bug, 6de491692d1992874fee06d15e040eb4d5da38c9 fixes the bug.